website/source/docs/install/bootstrapping.html.md

```diff
@@ -0,0 +1,108 @@
+---
+layout: "docs"
+page_title: "Bootstrapping a Datacenter"
```

Comment:
I know we already discussed the cluster vs datacenter thing, but this doc mostly refers to a cluster rather than a datacenter. I think it would make more sense to say cluster here.

Reply:
I kept the title from the original guide, so I'm going to defer this discussion to Eng.